src/sgame/sg_utils.cpp

```diff
@@ -945,3 +945,31 @@ glm::vec3 G_CalcMuzzlePoint( const gentity_t *self, const glm::vec3 &forward )
 	// Meh. I doubt it saves much. Casting to short ints might have, though. (copypaste)
 	return glm::floor( muzzlePoint + 0.5f );
 }
+
+//Dunno which ones are really useful, CBSE is not even self-contained.
+#include "sg_local.h"
```

Comment:
Don't include headers in the middle of the file

Reply:
original intent was to try to get rid those, but i'm feeling too lazy to dig in that hole. I'll move them on top.